### ISSUE TITLE
NoMethodError radar:fetch undefined method `[]' for nil:NilClass

### ISSUE THREAD

**Issue by bascht** [hacken-in/website#588]
Just go lang nothing to see here. Just some nil'ing. :)

```
lib/radar/meetup.rb:15:in `group_urlname': undefined method `[]' for nil:NilClass (NoMethodError)
	from lib/radar/meetup.rb:8:in `next_events'
	from lib/radar/base.rb:13:in `fetch'
	from app/models/radar_setting.rb:6:in `fetch'
	from lib/tasks/radar.rake:6:in `block (3 levels) in <top (required)>'
	from vendor/bundle/ruby/2.1.0/gems/activerecord-4.1.8/lib/active_record/relation/delegation.rb:46:in `each'
	from vendor/bundle/ruby/2.1.0/gems/activerecord-4.1.8/lib/active_record/relation/delegation.rb:46:in `each'
	from lib/tasks/radar.rake:5:in `block (2 levels) in <top (required)>'
	from vendor/bundle/ruby/2.1.0/gems/rake-10.3.2/lib/rake/task.rb:240:in `call'
	from vendor/bundle/ruby/2.1.0/gems/rake-10.3.2/lib/rake/task.rb:240:in `block in execute'
	from vendor/bundle/ruby/2.1.0/gems/rake-10.3.2/lib/rake/task.rb:235:in `each'
	from vendor/bundle/ruby/2.1.0/gems/rake-10.3.2/lib/rake/task.rb:235:in `execute'
	from vendor/bundle/ruby/2.1.0/gems/rake-10.3.2/lib/rake/task.rb:179:in `block in invoke_with_call_chain'
	from /usr/local/rvm/rubies/ruby-2.1.2/lib/ruby/2.1.0/monitor.rb:211:in `mon_synchronize'
	from vendor/bundle/ruby/2.1.0/gems/rake-10.3.2/lib/rake/task.rb:172:in `invoke_with_call_chain'
	from vendor/bundle/ruby/2.1.0/gems/rake-10.3.2/lib/rake/task.rb:165:in `invoke'
	from vendor/bundle/ruby/2.1.0/gems/rake-10.3.2/lib/rake/application.rb:150:in `invoke_task'
	from vendor/bundle/ruby/2.1.0/gems/rake-10.3.2/lib/rake/application.rb:106:in `block (2 levels) in top_level'
	from vendor/bundle/ruby/2.1.0/gems/rake-10.3.2/lib/rake/application.rb:106:in `each'
	from vendor/bundle/ruby/2.1.0/gems/rake-10.3.2/lib/rake/application.rb:106:in `block in top_level'
	from vendor/bundle/ruby/2.1.0/gems/rake-10.3.2/lib/rake/application.rb:115:in `run_with_threads'
	from vendor/bundle/ruby/2.1.0/gems/rake-10.3.2/lib/rake/application.rb:100:in `top_level'
	from vendor/bundle/ruby/2.1.0/gems/rake-10.3.2/lib/rake/application.rb:78:in `block in run'
	from vendor/bundle/ruby/2.1.0/gems/rake-10.3.2/lib/rake/application.rb:176:in `standard_exception_handling'
	from vendor/bundle/ruby/2.1.0/gems/rake-10.3.2/lib/rake/application.rb:75:in `run'
	from vendor/bundle/ruby/2.1.0/gems/rake-10.3.2/bin/rake:33:in `<top (required)>'
	from vendor/bundle/ruby/2.1.0/bin/rake:23:in `load'
	from vendor/bundle/ruby/2.1.0/bin/rake:23:in `<main>'
```

**Comment by bascht** [comment]
This concrete issue was caused by a faulty (?) user input that somehow made it to our radar settings.
E.g. 
```
2.1.2 :030 > RadarSetting.find(81)
 => #<RadarSetting id: 81, event_id: 373, url: "www.meetup.com/WordPress-Meetup-Koln", radar_type: "Meetup", last_processed: "2015-04-23 16:43:34", last_result: "undefined method `[]' for nil:NilClass">
```

There might be more of those nil-triggers around.
#590 only takes care of the meetup specific parts.

**Comment by moonglum** [comment]
Was fixed in #593
